### PR TITLE
Working mode proposal

### PR DESCRIPTION
**Pull request by cramforce**

### CODE REVIEW

**Comment by cramforce** [comment]
CC @tobie @mrjoro

**Review by paularmstrong — APPROVED** [review]

File: WORKING_MODE.md
Comment (rudygalfi) [inline WORKING_MODE.md]: spelling: Facilitators

**Comment by tobie** [comment]
For reference, [here](https://docs.google.com/document/d/10mzKsZ9zRllxkgRWNA5q2SDmj7P6Lajf2YPlJSwwj60/edit)'s the draft charter for the AC.

**Comment by tobie** [comment]
I'm curious why the Approvers WG is singled out in the work mode document. In what is that group different from the other WGs?

Maybe we should have a page where we list all the WGs and what each one of them is responsible for.

**Comment by mrjoro** [comment]
LGTM!

@tobie there's a [pending PR to document the Working Groups](https://github.com/ampproject/meta/pull/13).

**Comment by tobie** [comment]
Awesome. So I feel like the whole section about the Approvers WG should either be made generic (i.e. about all WGs) or be removed.

**Comment by davidstrauss** [comment]
> Awesome. So I feel like the whole section about the Approvers WG should either be made generic (i.e. about all WGs) or be removed.

I agree.

**Comment by sveronesi** [comment]
Perfect

File: WORKING_MODE.md
Comment (cramforce) [reply WORKING_MODE.md]: Fixed

**Comment by cramforce** [comment]
I definitely think the Approvers group is special in that it introduced a layer of indirection in the decision making process that isn't required by the governance doc; and the Approvers group is different from all other working group in that it's only job is decision making of work owned by another working group.

**Comment by davidstrauss** [comment]
I was partly agreeing with @tobie for the reasons he stated and partly because I've seen even approval working groups see significant refactoring even on mature projects (e.g. Fedora), making me reluctant to memorialize them more than necessary.

However, I hold this opinion very weakly. Please don't hold off on merging on my account.

File: WORKING_MODE.md
Comment (cramforce) [inline WORKING_MODE.md]: Call out specifically that we see this as status quo and something we'd adapt as we see fit

**Review by cpapazian — APPROVED** [review]

**Review by dvoytenko — APPROVED** [review]

File: WORKING_MODE.md
Comment (dvoytenko) [inline WORKING_MODE.md]: nit: link format is broken a bit with double parens?

**Review by dvoytenko — APPROVED** [review]

**Review by davidstrauss — APPROVED** [review]

File: WORKING_MODE.md
Comment (tobie) [inline WORKING_MODE.md]:
```suggestion
- Meeting notes are published in [this repository](https://github.com/ampproject/meta-tsc/labels/TSC%20Meeting).
```

Comment (cramforce) [reply WORKING_MODE.md]: Done

**Pull request merged**
